oak_containers_system_image/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+docker build . --tag oak-containers-system-image
+# We need to actually create a container, otherwise we won't be able to use `docker export` that gives us a filesystem image.
+# (`docker export` creates a tarball which has all the layers separate, which is _not_ what we want.)
+docker run oak-containers-system-image:latest
```

Comment:
do you need to use `docker run` here? Is it not possible to use docker create as in https://github.com/project-oak/oak/blob/main/oak_containers_container_example/build_container_bundle?

Reply:
If you look at Juliette's script, there's a `docker create` hidden in there.

However, indeed, using `docker create` is better than `docker run` as we don't really need to _run_ the container from the image we've built. Thanks for the link!